fontforge/python.c

```diff
@@ -6754,15 +6754,19 @@ return( -1 );
 	    else
 		last->next = cur;
 	    last = cur;
+	    if (uni > 0xFFFF)
+		bmp_compat = false;
 	}
     }
 
     AltUniFree(self->sc->altuni);
     self->sc->altuni = head;
 
     for ( fvs=self->sc->parent->fv; fvs!=NULL; fvs=fvs->nextsame ) {
-	fvs->map->enc = &custom;
-	FVSetTitle(fvs);
+	if ( !IsUnicodeEncoding(fvs->map->enc, bmp_compat) ) {
+	    fvs->map->enc = &custom;
+	    FVSetTitle(fvs);
+	} // XXX else rebuild map? 
```

Comment:
Is this this?

https://github.com/fontforge/fontforge/blob/9dff1f785b8fe1f0b7b7ea249705232a03ce4d2b/fontforge/encoding.c#L2073